addons/binding/org.openhab.binding.jeelink/ESH-INF/i18n/jeelink_de.properties

```diff
@@ -39,7 +39,7 @@ parameter.initdelay.description = Zeit nach der die Init Kommandos gesendet werd
 parameter.sensorid.label = Sensor ID
 parameter.sensorid.description = Die Sensor ID dieses Sensors.
 parameter.updateinterval.label = Update Intervall
-parameter.updateinterval.description = Das Update Intervall in Sekunden (0 setzt den Sensor in den 'live' Modus, d.h. jeder Wert wird an OpenHAB weitergereicht). Werte gr��er 0 machen nur Sinn, wenn eine Buffer Gr��e gr��er als 0 konfiguriert ist. 
+parameter.updateinterval.description = Das Update Intervall in Sekunden (0 setzt den Sensor in den 'live' Modus, d.h. jeder Wert wird an OpenHAB weitergereicht). Werte gr��er 0 machen nur Sinn, wenn eine Buffer Gr��e gr��er als 1 konfiguriert ist. 
```

Comment:
Could you create another pr to change all the occurrences to openHAB, so a lower case o